supertokens_python/async_to_sync_wrapper.py

```diff
@@ -14,20 +14,37 @@
 
 import asyncio
 from typing import Any, Coroutine, TypeVar
+from os import getenv
 
 _T = TypeVar("_T")
 
 
-def check_event_loop():
+def is_nest_asyncio_enabled():
     try:
-        asyncio.get_event_loop()
-    except RuntimeError as ex:
+        import nest_asyncio as _  # type: ignore
+
+        return getenv("SUPERTOKENS_NEST_ASYNCIO", "") == "1"
+    except Exception:
+        return False
+
+
```

Comment:
make sure that there is at least one thing in cicd where we run the python sdk (for example in e2e test with flask), where nest_asyncio is not a dependency and then we make sure it all works